### DISCUSSION TITLE
Omnibox base build feedback

### DISCUSSION THREAD

**Discussion by TheoBromine2022** [jon-harper/OmniBox#32] [General]
Okay! one week straight on .08mm layer height and I now have some really snazzy looking base prints!

Feedback: 
Make a note that the m4 screws joining the bases together will tap into the the rear base when inserted. 
When inserting the m3 nuts for the power terminal I would say that it really helps to use an m3x25mm to get the nut tightened into its recession by allowing you to poke the bolt through the hole and then snug the nut into place for the 16mms to go into after you remove the 25mm.
Zip tie ports, great idea but that first one next to the power terminal has a really sharp bend at the spade connection that I am not a fan of. Perhaps move the rear circular cutout for the zipties 40mm to the front of the omnibox? Nitpicky but I probably wont use that ziptie port so the change is negligible.

https://www.amazon.com/gp/product/B09SFQV4C7
https://www.amazon.com/gp/product/B08P9N2G7K

**Comment by jon-harper** [comment]
Thanks for the feedback! I really appreciate it.

> Make a note that the m4 screws joining the bases together will tap into the the rear base when inserted.

I will add this to the first assembly page, as this type of fastening is used throughout the case. Large entrance hole, smaller hole that the screw threads into.

> When inserting the m3 nuts for the power terminal I would say that it really helps to use an m3x25mm to get the nut tightened into its recession by allowing you to poke the bolt through the hole and then snug the nut into place for the 16mms to go into after you remove the 25mm.

I meant to make a note about this during the last build... I'll definitely include that. I will also see if I can make that corner more accessible.

> Zip tie ports, great idea but that first one next to the power terminal has a really sharp bend at the spade connection that I am not a fan of. Perhaps move the rear circular cutout for the zipties 40mm to the front of the omnibox? Nitpicky but I probably wont use that ziptie port so the change is negligible.

Agreed. This one is similar to the above: noticed during assembly and forgot to make an issue. Something for me to think about moving forward. :thinking:

**Comment by TheoBromine2022** [comment]
Started the main body yesterday after having a few weeks of other prints. Using the same settings as the base build its a 5 day print. Haha. Yeah.

**Reply by jon-harper** [reply]
Yikes! Are you going to stick with those settings or coarsen it up for the rest? I'm curious to see how it turns out!

**Reply by TheoBromine2022** [reply]
I'm rolling with it. Really smooth exterior on the base prints that I like. I might go coarser with a transparent petg for the next box I make on the ender5+ once I get this one done.

**Reply by jon-harper** [reply]
Very cool! Can you let me know how the PETG goes?

(P.S. I'm fond of using 0.16mm layer heights when I go for quality; it's a good balance of speed and layer height IMO.)

**Reply by TheoBromine2022** [reply]
I think my bridging settings are kinda off and I could use some more top layers for this print but it will be under the controller board so I won't see the defects anyway. So far the exterior looks great which is really all I care about. Yeah .08mm is really thin which is, well, slow. And is probably giving me little wisps of petg hairs that get stuck to the nozzle and make it messy. But I really do like the outcome. The base is done at that level, I'll post an image when I get home.

**Comment by TheoBromine2022** [comment]
![16656787464068312631839742448576](https://user-images.githubusercontent.com/111667850/195653943-ff7affae-7f3f-465e-90a6-96747a4aa032.jpg)

**Reply by jon-harper** [reply]
Very nice!
